genai-perf/genai_perf/export_data/telemetry_data_exporter_util.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+from typing import Dict, List, Optional
+
+from rich.console import Console
+from rich.table import Table
+
+
+class TelemetryDataExporterUtil:
```

Comment:
is there a reason why all these util functions are defined under `TelemetryDataExporterUtil` class? I think it's relatively common in the python world to define all the utility functions as just plain functions:
- https://github.com/google/jax/blob/main/jax/tools/build_utils.py
- https://github.com/pytorch/pytorch/blob/main/torch/_functorch/utils.py
- https://github.com/fastapi/fastapi/blob/master/fastapi/dependencies/utils.py

This avoids any need to care about `classmethod` or `staticmethod` (since the class attributes can just be global variables). Another benefit with this approach is we can abbreviate the long prefix when importing this utils module:
```python
from ... import telemetry_data_exporter_utils as telem_utils

telem_utils.export_telemetry_stats_csv(...)
```
instead of 
```python
from <...>.telemetry_data_exporter_utils import TelemetryDataExporterUtil

TelemetryDataExporterUtil.export_telemetry_stats_csv(...)
```
which IMO is less readable than the former.